app/scripts/modules/core/navigation/state.provider.ts

```diff
@@ -17,6 +17,10 @@ export class StateConfigProvider implements IServiceProvider {
   private root: INestedState = {
     name: 'home',
     abstract: true,
+    url: '?{debug:boolean}',
```

Comment:
whoops, I'm thinking of `bool` - this is the custom type I added, so, yeah, it does.